lib/refactorings/providers/open-ai-element-templates/OpenAIElementTemplatesProvider.js

```diff
@@ -220,4 +208,20 @@ function removeDuplicateRefactorings(refactorings) {
 
 function sortRefactoringsAlphabetically(refactorings) {
   return refactorings.slice().sort((refactoring, otherRefactoring) => refactoring.label.localeCompare(otherRefactoring.label));
+}
+
+export function idToToolName(id) {
+  return id.replace(/\./g, '_');
+}
+
+export function toolNameToId(name) {
+  return name.replace(/_/g, '.');
+}
+
+function getFunctionDescription(description, tags = []) {
+  if (tags.length) {
```

Comment:
As this is being read by the LLM I wonder if there is value in additional formatting.

Could be:

```
Description: "SOME DESCRIPTION"

Tags:

- A
- B
- C
- D

Reply:
I also wonder if we already `""` the description. Seems to not be the case, but _could_ be beneficial.

Reply:
I tried that and it certainly didn't make the results worse so let's roll with it.